### PR TITLE
added campaign model, containing url hash

### PR DESCRIPTION
**Pull request by laurieboyes**

### CODE REVIEW

File: sinks/models/campaign.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+module.exports = (message) => {
+
+	const campaign = {};
+	const url = message.egest.annotations.url;
+
+	if(url) {
+		campaign.hash = url.hash;
```
Comment (wheresrhys) [inline sinks/models/campaign.js]: I was thinking something more like
```
if (url.hash and /^#myft/.test(url.hash)) {
    campaign.myft = url.hash.replace(/^#myft\:/, '');
}
```
in beacon 2 trying to make properties as meaningful  to non-devs as possible

Comment (laurieboyes) [inline sinks/models/campaign.js]: Ahhhh

**Comment by wheresrhys** [comment]
:+1:

**Comment by wheresrhys** [comment]
sorry, merged by accident!

On 12 February 2016 at 15:44, Laurie Boyes <notifications@github.com> wrote:

> Merged #64 <https://github.com/Financial-Times/spoor-consumer/pull/64>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Financial-Times/spoor-consumer/pull/64#event-548684088>
> .
>

-- 

------------------------------

*This email was sent by a company owned by Financial Times Group Limited 
("FT Group <http://aboutus.ft.com/corporate-information/#axzz3rajCSIAt>"), 
registered office at Number One Southwark Bridge, London SE1 9HL.  
Registered in England and Wales with company number 879531. This e-mail may 
contain confidential information. If you are not the intended recipient, 
please notify the sender immediately, delete all copies and do not 
distribute it further.  It could also contain personal views which are not 
necessarily those of the FT Group.  We may monitor outgoing or 
incoming emails as permitted by law.*

**Pull request merged**